examples/runexamples.jl

```diff
@@ -21,6 +21,7 @@ include("LinearSystems.jl")
 include("SynchronousExamples.jl")
 include("ElectricalVehicleAndCharger.jl")
 include("CollidingBalls.jl")
+include("heattransfer2d.jl")
```

Comment:
I think this is the problem. The capitalization is wrong. It works locally under Windows because its file system is case insensitive. Travis tests are on Linux with a case-sensitive file system.

Reply:
Of course. I should have realized that.